client/post-editor/media-modal/test/markup.js

```diff
@@ -78,7 +78,7 @@ describe( 'markup', () => {
 
 			expect( value.type ).to.equal( 'dl' );
 			expect( ReactDomServer.renderToStaticMarkup( value ) ).to.equal(
-				'<dl class="wp-caption" style="width:276px;"><dt class="wp-caption-dt"><img src="https://s1.wp.com/wp-content/themes/a8c/automattic-2011/images/automattic-logo.png" alt="Automattic" width="276" class="alignnone size-full wp-image-1"/></dt><dd class="wp-caption-dd">Logo</dd></dl>'
+				'<dl class="wp-caption" style="width:276"><dt class="wp-caption-dt"><img src="https://s1.wp.com/wp-content/themes/a8c/automattic-2011/images/automattic-logo.png" alt="Automattic" width="276" class="alignnone size-full wp-image-1"/></dt><dd class="wp-caption-dd">Logo</dd></dl>'
```

Comment:
`width:276` is incorrect value, so it is going to be ignored by some browsers. Chrome:

![screen shot 2017-11-02 at 08 48 45](https://user-images.githubusercontent.com/699132/32315029-b9299dc2-bfaa-11e7-9296-db4fddfbb1ff.png)

Reply:
I think we need to fix `markup.caption` as part of React 16 migration instead.

Reply:
is it possible that `renderToStaticMarkup` behaves differently depending on its dom environment?

The code is using standard React inline styles to generate the width

Reply:
looks to be a difference between 15-->16 and not a difference in enzyme

Reply:
On investigation, I can't reproduce the error in the browser.  I haven't been able to figure out anything definitive yet, but when i manually caption an image in the post-editor I am seeing the "px" added:

<img width="1075" alt="screen shot 2017-11-06 at 3 22 24 pm" src="https://user-images.githubusercontent.com/4656974/32462317-f4b01e2a-c306-11e7-8124-7f09c0ba1455.png">

Reply:
Found an issue documenting whats happening: https://github.com/facebook/react/issues/1357.

In React 15 all css properties were assumed to be px units. In React 16 only integers are converted to px and all strings are left as-is.

Reply:
It looks like an incidental fix that surprised even core team members 😄 

I'm glad you tracked it down and fixed 👍